docker/dev/build/docker-entrypoint.sh

```diff
@@ -12,8 +12,8 @@ if [ "$1" = 'timesketch' ]; then
   cp /usr/local/src/timesketch/data/features.yaml /etc/timesketch/
   cp /usr/local/src/timesketch/data/tags.yaml /etc/timesketch/
   cp /usr/local/src/timesketch/data/ontology.yaml /etc/timesketch/
-  cp /usr/local/src/timesketch/data/sigma_config.yaml /etc/timesketch/
-  cp -r /usr/local/src/timesketch/data/sigma /etc/timesketch/sigma/
+  ln -s /usr/local/src/timesketch/data/sigma_config.yaml /etc/timesketch/sigma_config.yaml
```

Comment:
why this? and not a copy

or do we prefer a symbolic link for all files, not just those? so that changes to them are reflected immediately in the docker container?

Reply:
the idea here is that for the dev docker systems, it makes sense to have the mappings being used from the outside docker directory, so you can easier modify it, reload the server and you are good without the need to either open a shell or re-reploy. For sure we could also link to the other files the same way, but for this PR I am only concerned about the sigma stuff and here it would be the better solution.

But happy to make another PR to suggest also links to the others. WDYT?

Reply:
ok, let's keep it as is for now